### DISCUSSION TITLE
how to run code?

### DISCUSSION THREAD

**Discussion by maksim10411072** [krvajal/vscode-fortran-support#671] [Q&A - deprecated see Fortran Discourse]
![image](https://user-images.githubusercontent.com/99991427/191106594-06f09380-862d-4a84-865a-978b19296d6a.png)
where is compiler?

**Comment by gnikit** [comment]
On the released version there is no option to **compile** and **run** a single source file. On the development version, which is not yet released, we've added this option only for single files .

In general you can run code via the terminal similar to other compiled languages
